frontend/components/commons/sidebar/SidebarButton.vue

```diff
@@ -36,13 +35,23 @@ export default {
       required: true,
     },
     type: {
-      type: String,
-      required: true,
+      type: [String, null],
       validator: (value) => {
-        return ["Mode", "Metrics", "Refresh"].includes(value);
+        return ["Mode", "Metrics", "Refresh", "View-info", null].includes(value);
       },
     },
   },
+  computed: {
+    sidebarButtonClass() {
+      return [
+        this.type?.toLowerCase(),
+        this.activeView.includes(this.id) ? "active" : "",
+      ];
+    },
+    sidebarButtonTooltipText() {
+      return !this.activeView.includes(this.id) ? this.tooltip : null;
+    },
```

Comment:
Performance : this.activeView.includes(this.id) is done two times => you can add a computed to do the operation